src/domain/map/HankeMap.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import GeoJSON from 'ol/format/GeoJSON';
+import { Vector as VectorSource } from 'ol/source';
+import { Style, Fill, Stroke } from 'ol/style';
+import Map from '../../common/components/map/Map';
+import Controls from '../../common/components/map/controls/Controls';
+import LayerControl from '../../common/components/map/controls/LayerControl';
+import VectorLayer from '../../common/components/map/layers/VectorLayer';
+import Kantakartta from './Layers/Kantakartta';
+import DataLayers from './Layers/DataLayers';
+import Ortokartta from './Layers/Ortokartta';
+import styles from './Map.module.scss';
+import { useMapDataLayers } from './hooks/useMapDataLayers';
+import { MapDataLayerKey } from './types';
+
+type Props = {
+  hankeTunnus: string | undefined;
+};
+
+const HankeMap: React.FC<Props> = ({ hankeTunnus }) => {
+  console.log(hankeTunnus);
+  const { dataLayers, toggleDataLayer } = useMapDataLayers();
+
+  const [zoom] = useState(0);
+  const [showKantakartta, setShowKantakartta] = useState(true);
+  const [showOrtokartta, setShowOrtokartta] = useState(false);
+
+  // Temporary reference style implementation. Actual colors
+  // are chosen based on törmäysanalyysi
+  const geometryStyle = {
```

Comment:
geometryStyle and sampleHankeGeometry objects should be moved to outside of component. Otherwise react will rerender child component on every render cycle.